instructors.md

```diff
@@ -105,117 +105,28 @@ to make it easier for scientists to find what they want to.
 
 ## Resources
 
-*   `code/gen-survey-database.sql`: re-generate survey database used in examples.
-*   `code/sqlitemagic.py`: IPython Notebook plugin to handle SQLite databases.
 *   `data/*.csv`: CSV versions of data in sample survey database.
+*   `tools/create-db.sql`: generate survey database used in examples based on CSV.
+*   `tools/sqlitemagic.py`: IPython Notebook plugin to handle SQLite databases.
 
 ## SQLite Setup
 
 In order to execute the following lessons interactively,
-please install SQLite as mentioned in the setup instructions for your workshop,
-then create a directory `swc_sql`:
+please install SQLite as mentioned in the setup instructions for your workshop.
+Then
 
 ~~~ {.bash}
-$ mkdir swc_sql
-$ cd swc_sql
+$ git clone http://github.com/swcarpentry/sql-novice-survey.git
```

Comment:
Actually, I see now that **I** made a PR long ago (#83) to use the zip file similarly to the other lessons :)